extension/src/popup/views/SignTransaction/index.tsx

```diff
@@ -200,24 +201,24 @@ export const SignTransaction = () => {
     );
   }
 
-  if (!isHttpsDomain && !isExperimentalModeEnabled) {
-    return (
-      <WarningMessage
-        handleCloseClick={() => window.close()}
-        isActive
-        variant={WarningMessageVariant.warning}
-        header={t("WEBSITE CONNECTION IS NOT SECURE")}
-      >
-        <p>
-          <Trans domain={domain}>
-            The website <strong>{{ domain }}</strong> does not use an SSL
-            certificate. For additional safety Freighter only works with
-            websites that provide an SSL certificate.
-          </Trans>
-        </p>
-      </WarningMessage>
-    );
-  }
+  // if (!isHttpsDomain && !isExperimentalModeEnabled) {
+  //   return (
+  //     <WarningMessage
+  //       handleCloseClick={() => window.close()}
+  //       isActive
+  //       variant={WarningMessageVariant.warning}
+  //       header={t("WEBSITE CONNECTION IS NOT SECURE")}
+  //     >
+  //       <p>
+  //         <Trans domain={domain}>
+  //           The website <strong>{{ domain }}</strong> does not use an SSL
+  //           certificate. For additional safety Freighter only works with
+  //           websites that provide an SSL certificate.
+  //         </Trans>
+  //       </p>
+  //     </WarningMessage>
+  //   );
+  // }
```

Comment:
Should we remove this?

Reply:
whoops! This was commented out to test an API path locally, commented back in https://stellarfoundation.slack.com/archives/C06GA82KU2H/p1707081896826859?thread_ts=1706810112.655399&cid=C06GA82KU2H